tests/utils/test_utils.py

```diff
@@ -104,7 +104,7 @@ def test_validate_query_format_missing_placeholder(mocker):
     query_string = "This is a query with placeholders {placeholder1} and {placeholder2}"
     expected_placeholders = {"placeholder1", "placeholder2", "placeholder3"}
     validate_query_format(query_string, expected_placeholders)
-    mock_log_error.assert_called_with('Missing placeholders: %s, Extra placeholders: %s.\n For the query: %s', {'placeholder3'}, set(), 'This is a query with placeholders {placeholder1} and {placeholder2}')
+    mock_log_error.assert_called_with('%s%s\n For the query: %s', "Missing placeholders: {'placeholder3'}. ", '', 'This is a query with placeholders {placeholder1} and {placeholder2}')
```

Comment:
```suggestion
    mock_log_error.assert_called_with('%s%s\nFor the query: %s', "Missing placeholders: {'placeholder3'}. ", '', 'This is a query with placeholders {placeholder1} and {placeholder2}')
```

Reply:
Logging message changed in commit: [dfd74c6](https://github.com/AbsaOSS/living-doc-generator/pull/41/commits/dfd74c68b18f2569dc037e92c10c682092dbb155). All other comments in this section are also done with this commit.